{
  "failed_documents": [
    {
      "name": "PTK-C1T1C2T2",
      "version": "0.0.3",
      "url": "https://github.com/pamgene/PTK_C1T1C2T2_template",
      "error": "ServiceError({statusCode: 404, error: release.not.found, reason: Release for tagName 0.2.1 not found in https://github.com/tercen/vsn_operator})"
    },
    {
      "name": "PTK-CT1T2T3",
      "version": "0.0.3",
      "url": "https://github.com/pamgene/PTK_C1T1T2T3_template",
      "error": "ServiceError({statusCode: 404, error: release.not.found, reason: Release for tagName 0.2.1 not found in https://github.com/tercen/vsn_operator})"
    },
    {
      "name": "STK_C1T1C2T2",
      "version": "0.0.2",
      "url": "https://github.com/pamgene/STK_C1T1C2T2_template",
      "error": "ServiceError({statusCode: 404, error: release.not.found, reason: Release for tagName 0.2.1 not found in https://github.com/tercen/vsn_operator})"
    },
    {
      "name": "STK_CT1T2T3",
      "version": "0.0.3",
      "url": "https://github.com/pamgene/STK_C1T1T2T3_template",
      "error": "ServiceError({statusCode: 404, error: release.not.found, reason: Release for tagName 0.2.1 not found in https://github.com/tercen/vsn_operator})"
    },
    {
      "name": "image_analysis_PTK_afterwash",
      "version": "0.0.8",
      "url": "https://github.com/pamgene/image_analysis_PTK_afterwash_template",
      "error": "ServiceError({statusCode: -142, error: task.test.operator.bad.value, reason: At column ds2.Mean_SigmBg -- val != refVal -- at line 0 -- val = 16.6615367077063 refVal = 136.102007469654})"
    },
    {
      "name": "image_analysis_PTK_prewash",
      "version": "0.0.7",
      "url": "https://github.com/pamgene/image_analysis_PTK_prewash_template",
      "error": "ServiceError({statusCode: -142, error: task.test.operator.bad.value, reason: At column ds2.Mean_SigmBg -- val != refVal -- at line 0 -- val = 16.6615367077063 refVal = 136.102007469654})"
    },
    {
      "name": "image_analysis_STK",
      "version": "0.0.7",
      "url": "https://github.com/pamgene/image_analysis_STK_template",
      "error": "ServiceError({statusCode: -142, error: task.test.operator.bad.value, reason: At column ds2.Mean_SigmBg -- val != refVal -- at line 0 -- val = 16.6615367077063 refVal = 136.102007469654})"
    }
  ],
  "notest_documents": [],
  "success_documents": [],
  "no_release_documents": []
}